src/cryptography/hazmat/primitives/ciphers/modes.py

```diff
@@ -139,6 +139,10 @@ def validate_for_algorithm(self, algorithm):
 @utils.register_interface(ModeWithAuthenticationTag)
 class GCM(object):
     name = "GCM"
+    # This is (2 ** 39) - 256 bits (see NIST GCM spec)
+    _MAX_ENCRYPTED_BYTES = 68719476704
+    # This is 2 ** 64 bits (see NIST GCM spec)
+    _MAX_AAD_BYTES = 2305843009213693952
```

Comment:
Any reason not to just write the constants as expressions, so we can save the comment?